src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -61,6 +60,15 @@ namespace application_manager_test {
 namespace {
 const uint32_t kCorrelationId = 1;
 const uint32_t kAppId = 123;
+#ifdef OS_WINDOWS
+const std::string full_file_path = "C:\\file.txt";
+const std::string relative_file_path = "file.txt";
+const std::string app_storage_folder = "";
+#else
+const std::string full_file_path = "/file.txt";
+const std::string relative_file_path = "./file.txt";
+const std::string app_storage_folder = "./";
+#endif
```

Comment:
@Kozoriz Under Linux you have no write access to `/` and the `full_file_path` cannot be created or modified - unlike Windows. I would propose you to consider new value of  `full_file_path`